src/style.css

```diff
@@ -1,3 +1,67 @@
+html{
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  --light-blue: #00ACDC;
```

Comment:
Better not to use color name in variable names
Better to name it '--hover-color'